sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_container_registry_client.py

```diff
@@ -46,12 +46,13 @@ def __init__(self, endpoint: str, credential: Optional["AsyncTokenCredential"] =
                 :language: python
                 :dedent: 8
                 :caption: Instantiate an instance of `ContainerRegistryClient`
-        """
+        """       
+        defaultScope = [audience + "/.default"]
         if not endpoint.startswith("https://") and not endpoint.startswith("http://"):
             endpoint = "https://" + endpoint
         self._endpoint = endpoint
         self._credential = credential
-        super(ContainerRegistryClient, self).__init__(endpoint=endpoint, credential=credential, **kwargs)
+        super(ContainerRegistryClient, self).__init__(endpoint=endpoint, credential=credential, credential_scopes=defaultScope, **kwargs)
```

Comment:
My understanding is that Python wasn't going to use the `credential_scopes` kwarg in this library at all.  It would be worth following up with Johan, Laurent, or Anna on this as an implementation detail.

Reply:
Yes, `credential_scopes` has been removed from kwarg in client class in this PR. The `credential_scopes` here is a kwarg in base class, just for internal use. I've talked with @annatisch , it's okay to use it internally.

@annatisch Please correct me if I misunderstood.

Reply:
Is this a concern about the logic/mechanics around using 'credential_scope'? Or just the parameter name?
If the latter, then this is an internal implementation detail that could be refactored at any time and is not surfaced to the public API in any way.
If the former, please provide more details :)

Reply:
My concern was the latter.  There was a conversation between Johan and Laurent about whether to expose the `credential_scopes` kwarg, which I understand Python is trying to sunset, in favor of the `audience` kwarg.  If this implementation doesn't put that decision in danger, then I'm not worried at all.